pkg/opssight/perceivers.go

```diff
@@ -194,6 +302,50 @@ func (p *SpecConfig) ImagePerceiverService() *components.Service {
 	return p.perceiverService(p.names["image-perceiver"])
 }
 
+// ArtifactoryPerceiverService creates a service for the Artifactory perceiver
+func (p *SpecConfig) ArtifactoryPerceiverService() *components.Service {
+	name := p.names["artifactory-perceiver"]
+	service := components.NewService(horizonapi.ServiceConfig{
```

Comment:
not reusing perceiverService Method?

Reply:
I chose this to differentiate ports but I think I don't even need a service. I'm testing this rn, meanwhile you can focus on the rest of the stuff. I'll fix this